xl-op/blueprint.yaml

```diff
@@ -551,7 +551,7 @@ spec:
       overrideDefault: true
       prompt: !expr "IsCustomImageRegistry ? 'Enter the operator image to use (eg: <imageRegistryName>/<repositoryName>/<imageName>:<tagName>):' : 'Enter the operator image to use (eg: <repositoryName>/<imageName>:<tagName>):'"
       promptIf: !expr ServerType == 'dai-release' && (ProcessType == 'install' || ProcessType == 'upgrade')
-      default: xebialabsunsupported/release-operator:23.3.0-819.1130
+      default: xebialabsunsupported/release-operator:23.3.0-906.858
```

Comment:
maybe this is not intended to be changed. and is left as per master branch update.

Reply:
Updated to the latest master image build